assets/src/scripts/components/hydrograph/index.js

```diff
@@ -236,6 +236,7 @@ const createTitle = function(elem) {
 const watermark = function (elem) {
     elem.append('img')
         .classed('watermark', true)
+        .attr('alt', 'USGS logo')
```

Comment:
This is a nit, but I think this should represent what's on the logo.. perhaps: "USGS - science for a changing world".

Reply:
I'm cool with that.